streams/src/test/java/org/apache/kafka/streams/state/internals/FilteredCacheIteratorTest.java

```diff
@@ -127,4 +132,56 @@ public void shouldThrowUnsupportedOperationExeceptionOnRemove() {
         allIterator.remove();
     }
 
+    public static class KeyValueStoreWrapper<K extends Comparable<K>, V> {
```

Comment:
I think I'd recommend renaming this to something more accurate (i.e., it's not a store wrapper), and elevating it to its own class so it's available for other tests.

Reply:
:+1: Either way. In another comment, I pointed out that other tests could be simplified by using this class instead of implementing a special serde and using the bytes store. In that case, it would be used by other tests and would need to be in its own class.

But, yes, if it stays in this test only, then it should be private.